### PR TITLE
Multiple rooms

### LINKED ISSUE #4: support multiple rooms?
Could be nice if it was in multiple rooms?

### PR DESCRIPTION
**Pull request by SolarDrew**
Add tentative support for multiple matrix rooms. Listens for messages in all rooms specified in the config and responds to received messages in the same room.

Closes #4 .

### CODE REVIEW

**Comment by Cadair** [comment]
This seems to have a bunch of bonus commits?

**Comment by Cadair** [comment]
This seems to have a bunch of bonus commits?

**Comment by SolarDrew** [comment]
Yes, I noticed that and I have no idea what they're there for. I guess I'll rebase it? Is that a thing that fixes this?

**Comment by Cadair** [comment]
yeah, probably best way.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,12 @@ connectors:
     # Required
     mxid: "@username:matrix.org"
     password: "mypassword"
+    # Name of a single room to connect to
     room: "#matrix:matrix.org"
+    # Alternatively, a dictionary of multiple rooms
+    # One of these should be named 'main'
+    rooms: {'main': '#matrix:matrix.org',
```
Comment (Cadair) [inline README.md]: this can be written as:
```
rooms:
  main: '#matrix:matrix.org'
  other: '#riot:matrix.org'
```

**Pull request merged**